java/src/main/java/com/avatao/tfw/tfwconnector/Utilities.java

```diff
@@ -0,0 +1,30 @@
+package com.avatao.tfw.tfwconnector;
+
+import com.avatao.tfw.tfwconnector.TFWServerConnector;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
+public class Utilities {
+    private TFWServerConnector serverConnector;
+
+    public Utilities(TFWServerConnector serverConnector) {
+        this.serverConnector = serverConnector;
+    }
+
+    /**
+     * Send a trigger to the FSM to activate a transition
+     * @param message transition you want to trigger
+     */
+    public void sendTrigger(String message) {
```

Comment:
Parameter `String message` should be something like `String trigger` (this is not a message, but an FSM transition)